CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * **[Fix]** Remove old support libraries for compatibility with apps without enabled Jetifier tool. 
 * **[Feature]** Add a `AppCenter.setNetworkRequestsAllowed(bool)` API to block any network requests without disabling the SDK.
+* **[Feature]** Improved token encryption algorithm.
```

Comment:
I would tell more here, like "Add Encrypt then Mac to the existing AES crypto algorithm".